src/main/kotlin/com/exactpro/th2/common/message/MessageUtils.kt

```diff
@@ -517,7 +517,7 @@ fun Message.toTreeTable(): TreeTable = TreeTableBuilder().apply {
 val MessageIDOrBuilder.isValid: Boolean
     get() = bookName.isNotBlank()
             && hasConnectionId() && connectionId.sessionAlias.isNotBlank()
-            && hasTimestamp() && timestamp.seconds > 0 && timestamp.nanos > 0
+            && hasTimestamp() && timestamp.seconds > 0
```

Comment:
We must consider the case when seconds = 0 but nanos is not
```suggestion
            && hasTimestamp() && (timestamp.seconds > 0 || timestamp.nanos > 0)
```